client/web/src/repo/blob/use-blob-store.ts

```diff
@@ -0,0 +1,30 @@
+import { EditorView } from '@codemirror/view'
+import create from 'zustand'
+
+import { lineScrollEnforcing, SelectedLineRange, setSelectedLines } from './codemirror/linenumbers'
+
+interface BlobStore {
+    editView: EditorView | null
```

Comment:
Does it make sense to keep the property name in sync with the type name to ease understanding?

```suggestion
    editorView: EditorView | null
```